mobify/progressive-web-scaffold
# Astro: Merlin's cart update badge

Fix to show the correct amount of cart badge in all tabs and respond to any cart changes in the checkout flow

## Changes
- created an App Event that triggers when `TabController`'s navigationView is triggered to update its cart count
- Listen in for any cart badge events in `cartModal` to correctly respond to any changes made in cart items (removing, editing number of items) 

## How to test-drive this PR
- add items to cart
- switch over to account tab. The badge now should have the correct amount
- open up checkout, remove an item
- close the checkout modal the counter badge should show the correct amount.
